src/List/ListItemContent.js

```diff
@@ -0,0 +1,38 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import Icon from '../Icon';
+
+class ListItemContent extends React.Component {
+    static propTypes = {
+        className: PropTypes.string,
+        icon: PropTypes.string,
+        subtitle: PropTypes.node,
+        useBodyClass: PropTypes.bool
+    };
+
+    static defaultProps = {
+        useBodyClass: false
+    };
+
+    render() {
+        const { children, className, icon, subtitle, useBodyClass, ...otherProps } = this.props;
+
+        const classes = classNames('mdl-list__item-primary-content', className);
+
+        const subtitleClassName = useBodyClass ? 'mdl-list__item-text-body' : 'mdl-list__item-subtitle';
+
+        return (
+            <span className={classes}>
+                {icon && <Icon name={icon} />}
```

Comment:
Hmm... I don't really want to create a specific component for that one, so I'll try something here...

Have `icon` and `avatar` as prop. (Both are exclusive, we cannot have an icon, and an avatar at the same time.
Both can take either a string (icon name), or an element. If it's a string, we create an Icon and attach the specific classname. If it's an element, we clone it and attach the classname as well.